tests/helpers/test_config_validation.py

```diff
@@ -320,6 +321,31 @@ def test_service():
     schema("homeassistant.turn_on")
 
 
+def test_action_type_schemas_support_alias():
+    """Test that all action type schemas allow alias field."""
+
+    def find_root_schema(node):
```

Comment:
Instead of this, we should just embed a config for each notification and error out if we don't have a config for a condition in `cv.ACTION_TYPE_SCHEMAS`.